### PR TITLE
docs: dockertest covid article

### PR DESCRIPTION
**Pull request by vinckr**
## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

File: src/pages/blog/ory-vs-covid.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+  Open source software critical for global virus response
+
+publishedAt: '2020-11-27'
+author: 'Vincent Kraus'
+overline: >
+  Dockertest - easy go tests with docker
+
+category: Article
+subtitle: >
+ How Open-source software and Ory can be part of a greater effort against the virus
```
Comment (zepatrik) [inline src/pages/blog/ory-vs-covid.md:20]: This was removed recently, adapt according to the other PRs :wink:

File: src/pages/blog/ory-vs-covid.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+Goverments worldwide use it to provide their citizens with a means of tracing infections or rather notifiying them of exposure to the virus. 
+Since most people either have an Apple or Android device this approach covers a broad chunk of the population. 
+
+Now what does this have to do with [Dockertest](https://github.com/ory/Dockertest)?
+
+Google's [open source reference implementation](https://github.com/google/exposure-notifications-server) of an notification server uses [Dockertest](https://github.com/ory/Dockertest) for [database testing](https://github.com/google/exposure-notifications-server/blob/c35c97708dec0a8a3e2ac54f89f9033be5da22a6/internal/database/database_util.go). 
+It is amazing to see a tool that was written to ease the development process at Ory is used by big tech organisations in the most impactful health operation in modern times. It shows what a reliable and essential tool [Dockertest](https://github.com/ory/Dockertest) has become over the years.
+
+Here you can see the full code of the implementation: 
+```
```
Comment (zepatrik) [inline src/pages/blog/ory-vs-covid.md:62]:
```suggestion
```go
```

Comment (zepatrik) [reply src/pages/blog/ory-vs-covid.md:62]: Or if the code is copied from GH, use https://github.com/ory/web/pull/75/files#diff-9cdb5dd0b1f36dc0efaf706b88cb1744be3b404f4095e8c4e384b51cd046ba94R128-R132

File: src/pages/blog/ory-vs-covid.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+	return db, &Config{
+		Name:     dbname,
+		User:     username,
+		Host:     container.GetBoundIP("5432/tcp"),
+		Port:     container.GetPort("5432/tcp"),
+		SSLMode:  "disable",
+		Password: password,
+	}
+}
+
```
Comment (zepatrik) [inline src/pages/blog/ory-vs-covid.md:177]:
```suggestion
```

**Review by zepatrik — COMMENTED** [review]
Just some small comments, I will read it thoroughly later.

File: src/pages/blog/ory-vs-covid.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+Now what does this have to do with [Dockertest](https://github.com/ory/Dockertest)?
+
+Google's [open source reference implementation](https://github.com/google/exposure-notifications-server) of an notification server uses [Dockertest](https://github.com/ory/Dockertest) for [database testing](https://github.com/google/exposure-notifications-server/blob/c35c97708dec0a8a3e2ac54f89f9033be5da22a6/internal/database/database_util.go). 
+It is amazing to see a tool that was written to ease the development process at Ory is used by big tech organisations in the most impactful health operation in modern times. It shows what a reliable and essential tool [Dockertest](https://github.com/ory/Dockertest) has become over the years.
+
+Here you can see the full code of the implementation: 
+```
+
+ NewTestDatabaseWithConfig creates a new database suitable for use in testing.
```
Comment (zepatrik) [inline src/pages/blog/ory-vs-covid.md:64]:
```suggestion
// NewTestDatabaseWithConfig creates a new database suitable for use in testing.
```

**Comment by vinckr** [comment]
It is now Ory Dockertest throughout the article. 
There could be some improvements to the headline/teaser and keywords.

File: src/pages/blog/ory-vs-covid.md [hunk 5]
```diff
@@ -0,0 +1,216 @@
...
+
+publishedAt: '2020-12-02'
+author: vinckr
+
+overline: |
+  Dockertest - Automated Database Testing in Docker   
+---
+
+
+### What is Ory Dockertest?
```
Comment (aeneasr) [inline src/pages/blog/ory-vs-covid.md:25]: Do not start posts directly with a headline or this happens:

Bildschirmfoto 2020-11-27 um 15 08 18

File: src/pages/blog/ory-vs-covid.md [hunk 6]
```diff
@@ -0,0 +1,216 @@
...
+This is where Docker and Ory Dockertest can help you create real databases for your tests. You dont have to set up a whole database infrastructure.  
+
+[Dockertest](https://github.com/ory/Dockertest) automates your interactions with the Docker API. This lets you test your services database integration in a simple and straightforward way. 
+You configure your specific test setup, create the databases you want to test, ping to ensure connection and after testing close and dispose of the database again.  
+This process becomes 
+
+In [Dockertest](https://github.com/ory/Dockertest) a pool represents a connection to the Docker API and is used to handle docker images and resources represent the docker containers.
+You can find examples for all common databases on [github.com/ory/Dockertest](https://github.com/ory/Dockertest). 
+
+### How Dockertest is helping keeping Covid in check
```
Comment (aeneasr) [inline src/pages/blog/ory-vs-covid.md:47]: This post has no H2 (`##`). Make sure to keep the structure of h2,h3,h4,h5, ...

https://yoast.com/how-to-use-headings-on-your-site/

**Review by aeneasr — CHANGES_REQUESTED** [review]

**Comment by aeneasr** [comment]
ready for merge?

**Comment by vinckr** [comment]
@aeneasr Headlines should be good now. 
Jared also gave his ok to publish. 👍

File: src/pages/blog/ory-vs-covid.md [hunk 7]
```diff
@@ -0,0 +1,215 @@
+---
+path: '/ory-vs-covid/'
```
Comment (aeneasr) [inline src/pages/blog/ory-vs-covid.md:2]:
```suggestion
path: '/dockertest-gaen-google-apple-exposure-notification-covid-19/'
```

File: src/pages/blog/ory-vs-covid.md [hunk 8]
```diff
@@ -0,0 +1,215 @@
...
+title: |
+  How Ory is helping keep Covid in check
+
+teaser: |
+  This article gives a short introduction to Dockertest and how Google is using it against Covid19,
+  also an overview about the open-source virus response in general. 
+
+seo:
+  title: |
+    Open source software critical for global virus response
```
Comment (aeneasr) [inline src/pages/blog/ory-vs-covid.md:13]:
```suggestion
    Write tests like Google and Apple Exposure Notifications using Dockertest
```

File: src/pages/blog/ory-vs-covid.md [hunk 9]
```diff
@@ -0,0 +1,215 @@
+---
+path: '/ory-vs-covid/'
+
+title: |
+  How Ory is helping keep Covid in check
```
Comment (aeneasr) [inline src/pages/blog/ory-vs-covid.md:5]:
```suggestion
  Keeping Covid-19 in check with ORY Dockertest
```

**Review by aeneasr — CHANGES_REQUESTED** [review]
Please replace all `Ory` with `ORY`. Replacing the naming is a concerted effort across all resources and needs to be done all at once.

**Comment by vinckr** [comment]
Ok changed it. 
The filename is correct like this?

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 10]
```diff
@@ -198,18 +197,18 @@ Open-source holds a massive potential of innovation and efficency, for all other
...
+As this blogpost documents, ORY works with a dedicated worldwide community assuring that developer friendly security tools, innovative product development in identity infrastructure, and advanced cloud deployment using cloud platform independent open source tools are broadly available.
 
 ### Conclusion 
 
-Thanks for taking the time to read this article! Check out (our blog)[https://www.ory.sh/blog] for more articles and tutorials.
+Thanks for taking the time to read this article! Check out (our blog)[https://www.ORY.sh/blog] for more articles and tutorials.
 
-Should you have further questions or feedback, visit the [ORY community forum](https://community.ory.sh/) or [chat](https://www.ory.sh/chat).
-[ORY Dockertest](https://github.com/ory/Dockertest) is open-source and freely available on [github](https://github.com/ory/kratos), please consider starring the repository.  
+Should you have further questions or feedback, visit the [ORY community forum](https://community.ORY.sh/) or [chat](https://www.ORY.sh/chat).
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:210]: Not the domain names...

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 11]
```diff
@@ -34,25 +34,24 @@ One such case is when you want to test integration of services with databases.
 When working with many mock database systems testing can be cumbersome because the database abstraction layer is often tied to one database type. 
 Using an API abstraction simplifies the process. But for small changes to the table structure you need to work on the mocks or deal with other ramifications in the API.  
 You have the problem that the mock is not a real database. You also have to rewrite the mock database and unit tests, when you make changes to your configuration.   
-This is where Docker and Ory Dockertest can help you create real databases for your tests. You dont have to set up a whole database infrastructure.  
+This is where Docker and ORY Dockertest can help you create real databases for your tests. You dont have to set up a whole database infrastructure.  
 
-[Dockertest](https://github.com/ory/Dockertest) automates your interactions with the Docker API. This lets you test your services database integration in a simple and straightforward way. 
+[Dockertest](https://github.com/ORY/Dockertest) automates your interactions with the Docker API. This lets you test your services database integration in a simple and straightforward way.
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:39]: Please be more careful when using cmd+replace. This should not have been replaced.

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 12]
```diff
@@ -34,25 +34,24 @@ One such case is when you want to test integration of services with databases.
...
 
-[Dockertest](https://github.com/ory/Dockertest) automates your interactions with the Docker API. This lets you test your services database integration in a simple and straightforward way. 
+[Dockertest](https://github.com/ORY/Dockertest) automates your interactions with the Docker API. This lets you test your services database integration in a simple and straightforward way. 
 You configure your specific test setup, create the databases you want to test, ping to ensure connection and after testing close and dispose of the database again.  
-This process becomes 
 
-In [Dockertest](https://github.com/ory/Dockertest) a pool represents a connection to the Docker API and is used to handle docker images and resources represent the docker containers.
-You can find examples for all common databases on [github.com/ory/Dockertest](https://github.com/ory/Dockertest). 
+In [Dockertest](https://github.com/ORY/Dockertest) a pool represents a connection to the Docker API and is used to handle docker images and resources represent the docker containers.
+You can find examples for all common databases on [github.com/ORY/Dockertest](https://github.com/ORY/Dockertest).
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:43]: same

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 13]
```diff
@@ -34,25 +34,24 @@ One such case is when you want to test integration of services with databases.
...
 
-[Dockertest](https://github.com/ory/Dockertest) automates your interactions with the Docker API. This lets you test your services database integration in a simple and straightforward way. 
+[Dockertest](https://github.com/ORY/Dockertest) automates your interactions with the Docker API. This lets you test your services database integration in a simple and straightforward way. 
 You configure your specific test setup, create the databases you want to test, ping to ensure connection and after testing close and dispose of the database again.  
-This process becomes 
 
-In [Dockertest](https://github.com/ory/Dockertest) a pool represents a connection to the Docker API and is used to handle docker images and resources represent the docker containers.
-You can find examples for all common databases on [github.com/ory/Dockertest](https://github.com/ory/Dockertest). 
+In [Dockertest](https://github.com/ORY/Dockertest) a pool represents a connection to the Docker API and is used to handle docker images and resources represent the docker containers.
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:42]: same

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 14]
```diff
@@ -34,25 +34,24 @@ One such case is when you want to test integration of services with databases.
...
+You can find examples for all common databases on [github.com/ORY/Dockertest](https://github.com/ORY/Dockertest). 
 
 ## How Dockertest is helping keeping Covid in check
 
 The (Google/Apple) Exposure Notification (GAEN) system was developed by both companies to help provide an effective framework for tracing Covid19 infections with an emphasis on privacy.
 Goverments worldwide use it to provide their citizens with a means of tracing infections or rather notifiying them of exposure to the virus. 
 Since most people either have an Apple or Android device this approach covers a broad chunk of the population. 
 
-Now what does this have to do with [Dockertest](https://github.com/ory/Dockertest)?
+Now what does this have to do with [Dockertest](https://github.com/ORY/Dockertest)?
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:51]: same

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 15]
```diff
@@ -34,25 +34,24 @@ One such case is when you want to test integration of services with databases.
...
-It is amazing to see a tool that was written to ease the development process at Ory is used by big tech organisations in the most impactful health operation in modern times. It shows what a reliable and essential tool [Dockertest](https://github.com/ory/Dockertest) has become over the years.
+Google's [open source reference implementation](https://github.com/google/exposure-notifications-server) of an notification server uses [Dockertest](https://github.com/ORY/Dockertest) for [database testing](https://github.com/google/exposure-notifications-server/blob/c35c97708dec0a8a3e2ac54f89f9033be5da22a6/internal/database/database_util.go). 
+It is amazing to see a tool that was written to ease the development process at ORY is used by big tech organisations in the most impactful health operation in modern times. It shows what a reliable and essential tool [Dockertest](https://github.com/ORY/Dockertest) has become over the years.
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:54]: same

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 16]
```diff
@@ -175,11 +174,11 @@ Health agencies around the world use this reference implementation of the Exposu
 The server receives and validates the coded exposure keys from positive cases and keys shared between mobile devices to determine if two devices were near each other.
 It then stores the keys in a database and if there is a corresponding match on a user's device, the app notifies the user that they have been exposed to a positive case.
 
-You can read more in depth about the server on [Googles github documentation](https://google.github.io/exposure-notifications-server/)
+You can read more in depth about the server on [Googles github documentation](https://google.github.io/exposure-notifications-server/).
 
 ## Open-source software drives innovative development
 
-The whole GAEN project is open-source, as is [Dockertest](https://github.com/ory/Dockertest). The whole concept of contact tracing or exposure notification works great with open source, since it cuts down complexity of the whole operation. 
+The whole GAEN project is open-source, as is [Dockertest](https://github.com/ORY/Dockertest). The whole concept of contact tracing or exposure notification works great with open source, since it cuts down complexity of the whole operation.
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:181]: same

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 17]
```diff
@@ -198,18 +197,18 @@ Open-source holds a massive potential of innovation and efficency, for all other
...
-As this blogpost documents, Ory works with a dedicated worldwide community assuring that developer friendly security tools, innovative product development in identity infrastructure, and advanced cloud deployment using cloud platform independent open source tools are broadly available.
+ORY promises to produce high quality open source software.   
+[Dockertest](https://github.com/ORY/Dockertest) is just one example of how ORY approaches advanced software development for cloud native applications.  
+As this blogpost documents, ORY works with a dedicated worldwide community assuring that developer friendly security tools, innovative product development in identity infrastructure, and advanced cloud deployment using cloud platform independent open source tools are broadly available.
 
 ### Conclusion 
 
-Thanks for taking the time to read this article! Check out (our blog)[https://www.ory.sh/blog] for more articles and tutorials.
+Thanks for taking the time to read this article! Check out (our blog)[https://www.ORY.sh/blog] for more articles and tutorials.
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:208]: same

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 18]
```diff
@@ -198,18 +197,18 @@ Open-source holds a massive potential of innovation and efficency, for all other
...
 This wont solve all - if any - problems of modern society, but it would be a great step towards transparency and in the long run trust. 
 
-### Dockertest is an integral part of the Ory Ecosystem
+### Dockertest is an integral part of the ORY Ecosystem
 
-Ory promises to produce high quality open source software.   
-[Dockertest](https://github.com/ory/Dockertest) is just one example of how Ory approaches advanced software development for cloud native applications.  
-As this blogpost documents, Ory works with a dedicated worldwide community assuring that developer friendly security tools, innovative product development in identity infrastructure, and advanced cloud deployment using cloud platform independent open source tools are broadly available.
+ORY promises to produce high quality open source software.   
+[Dockertest](https://github.com/ORY/Dockertest) is just one example of how ORY approaches advanced software development for cloud native applications.
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:203]: same

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md [hunk 19]
```diff
@@ -198,18 +197,18 @@ Open-source holds a massive potential of innovation and efficency, for all other
...
-[ORY Dockertest](https://github.com/ory/Dockertest) is open-source and freely available on [github](https://github.com/ory/kratos), please consider starring the repository.  
+Should you have further questions or feedback, visit the [ORY community forum](https://community.ORY.sh/) or [chat](https://www.ORY.sh/chat).
+[ORY Dockertest](https://github.com/ORY/Dockertest) is open-source and freely available on [github](https://github.com/ORY/kratos), please consider starring the repository.  
 It is free and helps grow the project and community.  
 
-Sign up to [the ORY newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53) and we will notify you of new updates to [ORY Dockertest](https://github.com/ory/Dockertest) and [other ORY projects](https://github.com/ory/).
+Sign up to [the ORY newsletter](https://ORY.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53) and we will notify you of new updates to [ORY Dockertest](https://github.com/ORY/Dockertest) and [other ORY projects](https://github.com/ORY/).
```
Comment (aeneasr) [inline src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:214]: same

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md (hunk 10 above)
Comment (aeneasr) [reply src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:210]:
```suggestion
Should you have further questions or feedback, visit the [ORY Community Forum](https://community.ory.sh/) or [ORY Slack](https://www.ory.sh/chat).
```

**Review by aeneasr — CHANGES_REQUESTED** [review]

File: src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md (hunk 11 above)
Comment (vinckr) [reply src/pages/blog/dockertest-gaen-google-apple-exposure-notification-covid-19.md:39]: Ok! 
I thought capitalization wouldnt matter in the URL.

**Review by aeneasr — APPROVED** [review]

**Pull request merged**